app/views/start/index.html.erb

```diff
@@ -8,3 +8,26 @@
   <li>skills and qualifications required</li>
   <li>tasks and hours</li>
 </ul>
+ 
+<%= form_tag("/select_soc_codes", method: "get") do %>
```

Comment:
Does this create a CSRF token and UTF8 field on the form? I don't think we want that for a form with method `get...`